spec/tc211/termbase_spec.rb

```diff
@@ -116,9 +116,6 @@
                     "status" => "identical",
                   },
                 ],
-                "authoritativeSource" => [
-                  { "link" => "https://www.iso.org/standard/20057.html" },
-                ],
```

Comment:
Why remove this? We do need authoritative source...?

Reply:
The authoritative sources is added under the sources with type authoritative [here](https://github.com/geolexica/tc211-termbase/blob/main/spec/tc211/termbase_spec.rb#L102-L109). 

Multiple authoritative sources were causing issues in glossarist, if the `authoritativeSource` comes after the sources array then it was either overriding the original sources or adding duplicate authoritative sources. So that is why I removed the separate key.

If we want to keep the separate `authoritativeSource` key then one solution can be to remove it from the sources array and update glossarist to merge both (sources and authoritativeSource) when reading concepts.

Reply:
@HassanAkbar got it. Thanks!